modules/frontend/traceid_handlers.go

```diff
@@ -58,7 +56,7 @@ func newTraceIDHandler(cfg Config, next pipeline.AsyncRoundTripper[combiner.Pipe
 			}
 		}
 
-		comb := combinerFn(o.MaxBytesPerTrace(tenant), marshallingFormat, traceRedactor)
+		comb := combinerFn(o.MaxBytesPerTrace(tenant), string(marshallingFormat), traceRedactor)
```

Comment:
we change the signature of the combinerFn and avoid casting it to string?

I assumed that we defined MarshallingFormat type so we can strongly type it? right?